lib/screen/main/main_screen.dart

```diff
@@ -20,61 +21,103 @@ class _MainScreenState extends ConsumerState<MainScreen> {
+                  const MainHeader(),
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      top: Dimens.space29,
+                      left: Dimens.space39,
                     ),
-                  ),
-                ),
-                const SizedBox(
-                  height: Dimens.space15,
-                ),
-                Flexible(
-                  child: Padding(
-                    padding: const EdgeInsets.symmetric(
-                      horizontal: Dimens.space39,
+                    child: Text(
+                      context.localization.title_products,
+                      textAlign: TextAlign.left,
+                      style: context.textTheme.titleMedium?.copyWith(
+                        color: ColorName.blueDiamondColor,
+                      ),
                     ),
-                    child: MasonryGridView.count(
-                      itemCount: 6,
-                      crossAxisCount: _mainGridCount,
-                      mainAxisSpacing: Dimens.space15,
-                      crossAxisSpacing: Dimens.space15,
-                      itemBuilder: (context, index) {
-                        return const MainItem();
-                      },
+                  ),
+                  const SizedBox(
+                    height: Dimens.space15,
+                  ),
+                  Flexible(
+                    child: Padding(
+                      padding: const EdgeInsets.symmetric(
+                        horizontal: Dimens.space39,
+                      ),
+                      child: MasonryGridView.count(
+                        itemCount: 6,
+                        crossAxisCount: _mainGridCount,
+                        mainAxisSpacing: Dimens.space15,
+                        crossAxisSpacing: Dimens.space15,
+                        itemBuilder: (context, index) {
+                          return const MainItem();
+                        },
+                      ),
                     ),
                   ),
+                  const SizedBox(
+                    height: Dimens.space15,
+                  ),
+                ],
+              ),
+            ),
+          ),
+          Expanded(
+            child: Row(
+              children: [
+                const VerticalDivider(
+                  width: 1,
+                  color: ColorName.whitePlatinumColor,
                 ),
-                const SizedBox(
-                  height: Dimens.space15,
-                ),
+                Flexible(child: _buildCartSection())
               ],
             ),
+          )
+        ],
+      ),
+    );
+  }
+
+  Widget _buildCartSection() {
+    return Container(
+      color: Colors.white,
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(
+              top: Dimens.space30,
+              left: Dimens.space39,
```

Comment:
Missing padding for the right?
```suggestion
              left: Dimens.space39,
              right: Dimens.space39,
```

Reply:
We don't really need the right padding but I will add it.